.github/workflows/deploy.yml

```
@@ -58,6 +58,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
```

```suggestion
          path: ${{ runner.temp }}/.buildx-cache
```

runner contextに`runner.temp`というやつがいるのでそれを使うと良いそうです
https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context

---

あれ、でもこれは消えると困るやつなんですっけ…？

---

https://github.com/docker/build-push-action/issues/132
公式とかでも/tmp/.buildx-cacheの方使ってるんですよね....
どっちがいいんでしょうか

---

確かに公式で使ってるやつもありますね https://docs.docker.com/build/ci/github-actions/examples/
あまり気にしなくてもいいのかも？